test/TesterInternal/CodeGeneratorTests.cs

```diff
@@ -21,7 +21,10 @@ public void ServiceInterface_IsGrainClass()
             Assert.False(TypeUtils.IsGrainClass(t), t.FullName + " is not grain class");
             t = typeof(Orleans.Runtime.GrainDirectory.RemoteGrainDirectory);
             Assert.False(TypeUtils.IsGrainClass(t), t.FullName + " should not be a grain class");
+#if !NETSTANDARD_TODO
+           //blocked by CachedTypeResolver not coreclr compatible yet
```

Comment:
if this compiles even if it doesn't pass the test, then it's better to have a failing test that we know we need fixing, otherwise it's hard to understand what is working and what is not